src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -85,67 +86,85 @@ const ResearchProfile = () => {
   ) : !person ? (
     <NotFound />
   ) : (
-    <BackgroundDiv>
-      <Helmet>
-        <title>{fullName}</title>
-      </Helmet>
-      <Typography variant="h1" sx={{ mt: '1rem', mb: '2rem' }}>
-        {fullName}
-      </Typography>
-      <Typography variant="h2">{t('common.employments')}</Typography>
-      {activeAffiliations.length > 0 ? (
-        <Box sx={lineSx}>
-          <WorkIcon />
-          <Box sx={textContainerSx}>
-            {activeAffiliations.map(({ organization }) => (
-              <AffiliationHierarchy key={organization} unitUri={organization} commaSeparated />
-            ))}
+    <Box>
+      <Box sx={{ bgcolor: 'person.main', py: '1.1rem', display: 'flex', alignItems: 'center', gap: '1rem' }}>
+        <Typography variant="h1" sx={{ ml: '2rem' }}>
+          {fullName}
+        </Typography>
+        {orcidUri && <img src={orcidIcon} height="20" alt="orcid" />}
+      </Box>
+      <BackgroundDiv>
+        <Helmet>
+          <title>{fullName}</title>
+        </Helmet>
+
+        {activeAffiliations.length > 0 ? (
+          <Box sx={lineSx}>
+            <Box sx={{ display: 'flex', gap: '0.5rem' }}>
+              {activeAffiliations.map(({ organization, role }) => (
+                <Box
+                  sx={{
+                    border: '2px solid',
+                    borderColor: 'secondary.light',
+                    borderRightColor: 'primary.main',
+                    borderRightWidth: '1px',
+                    width: 'fit-content',
+                    pr: '1.5rem',
+                    ':last-child': {
+                      border: 'none',
+                    },
```

Comment:
Er det en grunn til å sette border rundt alt her? Hvis vi endrer bakgrunnsfargene senere må vi da huske å endre fargen på border her også

```suggestion
                    pr: '1.5rem',
                    ':not(:last-child)': {
                      borderRight: '1px solid',
                      borderRightColor: 'primary.main',
                    },
```

Reply:
Om jeg ikke hadde border på hele når jeg først gjorde dette ville ikke border right rendre (på samme måte som jeg har gjort nå), men virker som om det var sliten pc og ikke koden som jeg først tenkte :|

Reply:
ser ut som denne fortsatt kan forenkles litt, om forslaget mitt ikke introduserer noen andre bugs?